app/views/plugins/templates/main.liquid

```diff
@@ -12,9 +12,7 @@
          {% include 'photos' %}
         </div>
 
-        {% if plugins.thermometer.active %}
-          {% include 'thermometer' %}
-        {% endif %}
+        {% include 'thermometer' %}
```

Comment:
@NealJMD sorry, just noticed you removed the active check. Will the toggle behaviour still work?